pilot/pkg/proxy/envoy/v2/debug_test.go

```diff
@@ -315,7 +315,7 @@ func TestAuthenticationZ(t *testing.T) {
 			name:           "dumps most recent proxy with 200",
 			proxyID:        "dumpApp-644fc65469-96dza.testns",
 			wantCode:       200,
-			expectedLength: 25,
+			expectedLength: 24,
```

Comment:
This change is because now I moved edsInc services out of standard init to the test, which creates a endpoint with ServiceAccount